profile/README.md

```diff
@@ -1 +1,12 @@
-Open Systems Pharmacology Suite manual: https://docs.open-systems-pharmacology.org
+<table>
+<tr>
+  <td><li><a href="http://setup.open-systems-pharmacology.org/">Latest suite release can be found here</a></li></td>
+  <td><li><a href="https://docs.open-systems-pharmacology.org">Check our documentation</a></li></td>
+  <td><li><a href="https://forum.open-systems-pharmacology.org">Need help? Visit the discussion forum</a></li></td>
+</tr>
+<tr>
+  <td><li><a href="https://www.open-systems-pharmacology.org/OSPSuite-R/">ospuite R package</a></li></td>
```

Comment:
Maybe this should be one of our main repo instead...instead of vision and mission :)

Reply:
>Maybe this should be one of our main repo instead...instead of vision and mission :)

You mean this repo? But it contains nothing except this short Readme...

Reply:
no i mean  the R stuff? weird to see components here
AT least it should be ospsuite-R ?